service/storage_proxy.cc

```diff
@@ -2574,7 +2582,24 @@ storage_proxy::storage_proxy(distributed<replica::database>& db, gms::gossiper&
 }
 
 struct storage_proxy::remote& storage_proxy::remote() const {
-    return *_remote;
+    if (_remote) {
```

Comment:
So now we will not be able to truncate local table before remote is available? BTW we should never sent RPC for ourselves. Truncate should be fixed.